src/pages/Leaderboard/useLeaderboard.ts

```diff
@@ -1,10 +1,14 @@
 import { useCallback, useEffect, useMemo, useState } from 'react';
 
+import { useTranslation } from 'react-i18next';
```

Comment:
❗️
Тут и везде нет смысла отбивать импорт одного внешнего пакет от другого
```suggestion
import { useTranslation } from 'react-i18next';
```

Reply:
Пофиксил